server/service/frontend_test.go

```diff
@@ -64,6 +64,5 @@ func TestServeEndUserEnrollOTA(t *testing.T) {
 	bodyBytes, err := io.ReadAll(response.Body)
 	require.NoError(t, err)
 	bodyString := string(bodyBytes)
-	require.Contains(t, bodyString, "Enroll your device to Fleet")
-	require.Contains(t, bodyString, "?enroll_secret=foo")
```

Comment:
I removed the assertion on the copy intentionally because I didn't see the real value of it, and the HTML content is more difficult to assert now.